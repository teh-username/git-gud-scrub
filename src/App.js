import React from 'react';
import FileSystem from './containers/FileSystem';
import FileStatus from './containers/FileStatus';
import CommitGraph from './containers/CommitGraph';
import Terminal from './containers/Terminal';

const App = () => [
  <FileSystem key="fs" />,
  <CommitGraph key="cg" />,
  <Terminal key="tl" />,
  <FileStatus key="ft" />,
];

export default App;
