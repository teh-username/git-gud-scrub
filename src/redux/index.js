import { combineReducers } from 'redux';
import fileSystem from './modules/fileSystem';

export default combineReducers({
  fileSystem,
});
