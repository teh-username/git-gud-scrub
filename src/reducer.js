import { combineReducers } from 'redux';
import fileSystem from './reducers/fileSystem';

export default combineReducers({
  fileSystem,
});
