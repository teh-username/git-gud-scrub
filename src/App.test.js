import React from 'react';
import ReactDOM from 'react-dom';
import { Provider } from 'react-redux';
import { createStore } from 'redux';

import App from './App';
import reducer from './redux/index';

it('renders as expected', () => {
  const wrapper = shallow(<App />);
  expect(wrapper).toMatchSnapshot();
});
