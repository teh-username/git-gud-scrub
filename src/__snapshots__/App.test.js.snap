// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders as expected 1`] = `
Array [
  <Connect(FileSystem)
    key="fs"
/>,
  <Connect(CommitGraph)
    key="cg"
/>,
  <Connect(Terminal)
    key="tl"
/>,
  <Connect(FileStatus)
    key="ft"
/>,
]
`;
